docs/src/devel/dev-env-setup.rst

```diff
@@ -178,3 +178,216 @@ e.g.:
 ::
 
    OS_MIGRATE_FUNC_TEST_ARGS='--tags test_network,test_subnet --skip-tags test_clean_after' ./toolbox/run make test-func
+
+Running e2e tests
+-----------------
+
+os-migrate also has a suite of end to end, e2e, tests which tests a migration from one existing openstack deployment
+to another existing openstack deployment.
+
+You can also test with a single existing openstack deployment migrating from one project to another.
+
+These docs cover the testing of a single existing openstack deployment migrating from one project to another scenario.
+
+The concepts and prerequisites are the same for other deployments.
+
+Prerequisites for e2e test
+--------------------------
```

Comment:
shouldn't this be a subsection of Running e2e tests ?

Reply:
same with the next subsections

Reply:
Subsection now more obvious using '^' as per other docs